_rules/html-xml-lang-match-5b7ae0.md

```diff
@@ -23,11 +23,10 @@ This rule applies to any [document element](https://dom.spec.whatwg.org/#documen
 
 ## Expectation
 
-For each test target, the value of the primary language subtag (characters before the first dash) for the `lang` and `xml:lang` attributes are the same.
+For each test target, the values of the [primary language subtags](https://tools.ietf.org/html/bcp47#section-2.2.1) and the [extended language subtags](https://tools.ietf.org/html/bcp47#section-2.2.2), if any exist, for the `lang` and `xml:lang` attributes are the same.
 
 ## Assumptions
 
-- Although there is no known way that an inappropriate secondary subtag in the `lang` or `xml:lang` attribute (such as `en-XYZ`) can lead to accessibility issues, it is conceivable that with specific assistive technologies on some languages, there can be exceptions. In that case `lang` and `xml:lang` should have matching secondary subtags.
```

Comment:
Given how long it took us to figure out that extended subtags are important, maybe we should replace that assumption with a note on the expectation?

Reply:
Based on code inspection and browser testing `xml:lang` is always ignored in Safari when calculating the language exposed through the Accessibility Tree (although `xml:lang` is used when calculating the CSS `:lang()` selector).

That means `lang` and `xml:lang` mis-match has no effect on accessibility in Safari.

The same looks to be true for Firefox and Chromium based on reading their code, but I still need to confirm in a browser (and also test IE11).

If this is confirmed then `lang` and `xml:lang` mis-match has no effect on accessibility in current user agents.

Reply:
Do we have any insights into AT behaviour as well?

Reply:
Here are the current results - results guessed from code still have '?' after them. Once confirmed in AT/Accessibility Inspector I've removed the  '?' 

https://github.com/act-rules/act-rules.github.io/issues/668#issuecomment-534606384

Reply:
I'll limit this pull request to only requiring that extended language subtags also match and then we can take a look at either extending this requirement to other subtags in the future.

Reply:
Let's continue the discussion here: https://github.com/act-rules/act-rules.github.io/pull/934/commits/dd90193a6b00afa9feec21351b7add501059ff4d#r328112033